include/ledmatrixgfx.h

```diff
@@ -82,6 +82,29 @@ class LEDMatrixGFX : public GFXBase
     {
         matrix.setBrightness(amount);
     }
+    
+    // EstimatePowerDraw
+    //
+    // Estimate the total power load for the board and matrix by walking the pixels and adding our previously measured
+    // power draw per pixel based on what color and brightness each pixel is/
+
+    int EstimatePowerDraw()
```

Comment:
I'm glad I got an out-of-band explanation of this, otherwise I don't think I would have followed. :)